insights/protocol.py

```diff
@@ -10,6 +10,9 @@
 MODEL_TYPE_FUNDS_FLOW = "funds_flow"
 MODEL_TYPE_FUNDS_FLOW_ID = 2
 
+MODEL_TYPE_BALANCE_TRACKING = "balance_tracking"
+MODEL_TYPE_BALANCE_TRACKING_ID = 3
```

Comment:
will we need model ids ? those can be now obsolete, same for type? please investigate

Reply:
it can be removed